src/controllers/selectedAccount/selectedAccount.ts

```diff
@@ -320,15 +320,43 @@ export class SelectedAccountController extends EventEmitter implements ISelected
     const walletORStkWalletToken = portfolioAccountState['1']?.result?.tokens.find(
       ({ address }) => address === STK_WALLET || address === WALLET_TOKEN
     )
-
+    const stkTokenInPortfolio = portfolioAccountState['1']?.result?.tokens.find(
+      ({ address }) => address === STK_WALLET
+    )
+    const stkBalanceUsd = stkTokenInPortfolio
+      ? Number(formatEther(stkTokenInPortfolio.amount)) *
+        (stkTokenInPortfolio.priceIn[0]?.price || 0)
+      : 0
```

Comment:
we will hurt the user's points if we do `price || 0`, because that will result in a `0` value snapshot when cena doesn't work.